_resources/e2e/postman/Vulnerability_DB_API.postman_collection.json

```diff
@@ -2517,6 +2517,74 @@
 					},
 					"response": []
 				},
```

Comment:
Can we include also tests for the Delete operations?

Reply:
Yes! Let me fix this.